src/SFA.DAS.QnA.Application.UnitTests/CommandsTests/ResetPageAnswersHandlerTests/When_page_found.cs

```diff
@@ -44,11 +44,10 @@ public async Task Then_questiontag_is_reset()
 
             var getApplicationDataResponse = await GetApplicationDataHandler.Handle(new GetApplicationDataRequest(ApplicationId), CancellationToken.None);
 
-            var applicationData = JObject.Parse(getApplicationDataResponse.Value);
+            var applicationData = JsonNode.Parse(getApplicationDataResponse.Value).AsObject();
             var questionTag = applicationData["Q1"];
 
-            questionTag.Should().NotBeNull();
-            questionTag.Value<string>().Should().BeNullOrEmpty();
+            questionTag.Should().BeNull();
```

Comment:
This test seems identical to the one in `ResetPageAnswersBySectionNoHandlerTests` -- please see my previous comment for that one

Reply:
done

Reply:
Since `questionTag` is a struct, it will never be null, so the assertion on 50 isn't doing anything. I think this test should assert that `applicationData` has only one property, and that its value is null. 

Right now the test will abort if the first assertion on line 50 fails, so you'll also have to find a way to make the test run both assertions. In NUnit you can wrap multiple asserts in an `Assert.Multiple()` block, but the FluentAssertions way will be different

Reply:
Sorry got confused with all the similar sounding names -- this one looks good now